lib/cosi/cosi.go

```diff
@@ -104,6 +104,14 @@ type Signature struct {
 	Response  abstract.Secret
 }
 
+// NewSignature returns a pre-initialized signature with Secret.One
+func NewSignature(s abstract.Suite) *Signature {
```

Comment:
Mhhh. Well for me it seems strange to send a "nil" `SkipBlock` that gets filled by other peers, why not send a request to get the block directly ? But that's a separate issue - 
In the meantime, yes, that'd be cleaner to have that in `skipchain/struct.go` as a private method. Thanks.